src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(number) {
+      calculator.result += number;
+
+      return this;
+    },
+    subtract(number) {
+      calculator.result -= number;
+
+      return this;
+    },
+    multiply(number) {
+      calculator.result *= number;
+
+      return this;
+    },
+    divide(number) {
+      calculator.result /= number;
```

Comment:
There is a potential issue with dividing by zero in the `divide` method. Consider adding a check to prevent division by zero, which could cause an error or unexpected behavior.